Common/Core/aodMerger.cxx

```diff
@@ -48,10 +50,13 @@ int main(int argc, char* argv[])
     } else if (c == 2) {
       maxDirSize = atol(optarg);
     } else if (c == 3) {
+      skipNonExistingFiles = (strcmp(optarg, "true")) ? false : true;
+    } else if (c == 4) {
       printf("AOD merging tool. Options: \n");
-      printf("  --input <inputfile.txt>      Contains path to files to be merged. Default: %s\n", inputCollection.c_str());
-      printf("  --output <outputfile.root>   Target output ROOT file. Default: %s\n", outputFileName.c_str());
-      printf("  --max-size <size in Bytes>   Target directory size: %ld \n", maxDirSize);
+      printf("  --input <inputfile.txt>                  Contains path to files to be merged. Default: %s\n", inputCollection.c_str());
+      printf("  --output <outputfile.root>               Target output ROOT file. Default: %s\n", outputFileName.c_str());
+      printf("  --max-size <size in Bytes>               Target directory size. Default: %ld\n", maxDirSize);
+      printf("  --skip-non-existing-files <true|false>   Non-existing files are skipped. Default: %s\n", "false");
```

Comment:
I would not put an argument to this. The default is false and then one just provides `--skip-non-existing-files` if one wants this option. What do you think?